game-core/gradle.properties

```diff
@@ -1,3 +1,2 @@
-checkstyleIntegTestMaxWarnings=0
```

Comment:
You're going to have to update the _.travis/update_checkstyle_thresholds_ script to no longer write this value.